bin/init.js

```diff
@@ -34,7 +34,7 @@ module.exports = () => {
 		})
 }
 
-const setProjectID = () => {
+exports.setProjectID = () => {
```

Comment:
What's the benefit of using exports here?

Reply:
will be adding more testing coverage in the future.  can't test private methods cleanly.  nothing dangerous is exposed here